README.md

```diff
@@ -70,14 +70,14 @@
 > 
 > - Reference: Modify settings menu
 > 
-### :star: [TOH+](https://github.com/SkullCreeper/TownOfHostPlus)
+### :star: [TOH+](https://github.com/ItzLoonie/TownOfHostPlus)
```

Comment:
WHAT 
![image](https://github.com/user-attachments/assets/50a6aba9-2e25-458a-8d9f-02798e07c012)

Reply:
wait, these are probably not even used now due to the new update re-hauling the entire thing